x-pack/plugins/apm/common/time_range_metadata.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface TimeRangeMetadata {
```

Comment:
Could this be moved in ./public?

Reply:
The intent was to use it on the server as well, but I forgot about it - thanks for pointing it out, I've pushed a change that adds it to the endpoint definition as well.